src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[5];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = new FigureSupplier().getRandomFigure();
```

Comment:
checklist : "You should create several random Figures, so you will use a loop. Please don’t create a new FigureSupplier() inside the loop."